exporters/otlp/src/main/java/io/opentelemetry/exporter/otlp/OtlpGrpcSpanExporter.java

```diff
@@ -111,8 +112,7 @@ private OtlpGrpcSpanExporter(ManagedChannel channel, long deadlineMs) {
    */
   @Override
   public CompletableResultCode export(Collection<SpanData> spans) {
-    spansSeen.add(
-        spans.size(), Labels.of("exporter", OtlpGrpcMetricExporter.class.getSimpleName()));
+    spansSeen.add(spans.size(), Labels.of("exporter", OtlpGrpcSpanExporter.class.getSimpleName()));
```

Comment:
Thanks for the fixes! While we're here, should we extract constants for constant labels?

Reply:
Though as an aside, after seeing some metrics code recently, I sort of wish constant labels could be specified when creating a meter instead of only when observing to save the tedium of always extracting constants to save allocation

Reply:
This was a feature in the API that we decided to remove from the API for now. It may very well come back later, but we wanted to limit the features pre-GA, I think.

Reply:
yeah, I guess this could just be a constant for the labels instance, eh? Very happy to make that happen, either as a follow-up, or part of this PR.

Reply:
Either's fine with me this is approved :)

Reply:
easy enough to do, so I just amended this PR with the constant label extraction